Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "at_rust"
-version = "0.2.0"
+version = "0.2.1"
```

Comment:
We changed to v0.2.1, but the changelog is still at v0.2.0

Reply:
Thanks @XavierChanth that's taken care of now